src/compiler/cbs/PipelineBuilder.cpp

```diff
@@ -70,6 +70,10 @@ void registerCBSPipelineLegacy(llvm::legacy::PassManagerBase &PM) {
 }
 #endif // LLVM_VERSION_MAJOR < 16
 
+#if defined(ROCM_CLANG_VERSION_MAJOR) && ROCM_CLANG_VERSION_MAJOR == 5 && ROCM_CLANG_VERSION_MINOR == 5 && ROCM_CLANG_VERSION_PATCH == 0
```

Comment:
I think this is too restrictive. I don't remember ROCm ever pulling in upstream updates for their compiler version in a patch version... so I'd rather just check major and minor. here..

Reply:
```suggestion
#if defined(ROCM_CLANG_VERSION_MAJOR) && ROCM_CLANG_VERSION_MAJOR == 5 && ROCM_CLANG_VERSION_MINOR == 5
```